pkg/istio/servicerolebinding/servicerolebinding.go

```diff
@@ -19,7 +21,8 @@ import (
 )
 
 const (
-	allUsers = "user.*"
+	allUsers         = "user.*"
+	wildCardAllUsers = "*"
```

Comment:
Does istio have a constant for this wildcard use case which we could use?

Reply:
Couldn't find exportable constant for wildcard, let me know if you found one